### DISCUSSION TITLE
note-ml2024fall/tree-models-and-ensemble-learning/

### DISCUSSION THREAD

**Discussion by giscus** [Cgfyufsygsm/cgfyufsygsm.github.io#72] [Announcements]
# note-ml2024fall/tree-models-and-ensemble-learning/

本文主要涉及机器学习中的树模型以及集成学习。 一键回城。   树模型  决策树 回忆：我们之前的很多模型都可以写成 y=wTx+by = w^Tx+by=wTx+b，决策逻辑（二分类）：f(x)≥0f(x)\ge0f(x)≥0 时预测为 111，否则预测为 000。 事实上这个逻辑可以写成一个树。  我们之前学习的模型多为线性模型，为了应对非线性性，我们可以考虑引入决策树。 决策树定义：一棵包含

https://blog.imyangty.com/note-ml2024fall/tree-models-and-ensemble-learning/

**Comment by hecatolite** [comment]
跪求更新

**Reply by Cgfyufsygsm** [reply]
在更了在更了🥺🥺🥺🥺大概今晚六七点前肯定可以上线
